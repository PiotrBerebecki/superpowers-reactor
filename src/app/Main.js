import React from 'react';

import { MainWrapper, MainContainer } from './../styled';

function Main({ children }) {
  return (
    <MainWrapper>
      <MainContainer>
        {children}
      </MainContainer>
    </MainWrapper>
  );
}

export default Main;
