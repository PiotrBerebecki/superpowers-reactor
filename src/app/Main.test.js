/**
 * @jest-environment node
 */

import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { MemoryRouter } from 'react-router-dom';
import 'jest-styled-components';

import Main from './Main';

describe('Main', () => {
  it('renders', () => {
    const wrapper = shallow(<Main />);
    expect(shallowToJson(wrapper)).toMatchStyledComponentsSnapshot();
  });
});
