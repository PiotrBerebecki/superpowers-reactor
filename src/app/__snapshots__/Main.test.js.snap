// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders 1`] = `
<styled.main>
  <styled.section />
</styled.main>
`;
